index.html

```diff
@@ -3,10 +3,5 @@
 title: Beyond Melee
 ---
 <head>
-    <meta http-equiv = "refresh" content = "4; url = ./home/" />
-</head>
-<body class="black-bg">
-    <video autoplay muted id="introVideo" poster="bm-logo-transparent.png">
-        <source src="/videos/title.mp4" type="video/mp4"/>
-    </video>
-</body>
+    <meta http-equiv = "refresh" content = "url=/home/" />
```

Comment:
Intro video is unnecessary in my opinion.

Reply:
this should be in a separate PR since it’s not about cleaning things up and is instead about your personal taste

Reply:
sorry, i've gotten 2 compliments about the vid and 0 complaints so far, i'm keeping it